tests/models/encoder_decoders/wikitables_semantic_parser_test.py

```diff
@@ -0,0 +1,64 @@
+# pylint: disable=invalid-name
+from typing import List
+
+import tqdm
+
+from allennlp.common.testing import ModelTestCase
+from allennlp.common import Params
+from allennlp.data import Dataset, DatasetReader, Instance, Token
+from allennlp.data.fields import Field, LabelField, ListField, TextField
+from allennlp.data.tokenizers import CharacterTokenizer
+from allennlp.data.token_indexers import SingleIdTokenIndexer
+from allennlp.data.dataset_readers.seq2seq import START_SYMBOL, END_SYMBOL
+
+
+@DatasetReader.register("toy_wikitables")
+class ToyWikiTablesDatasetReader(DatasetReader):
```

Comment:
Do you plan to use the actual `WikitablesDatasetReader` for tests when it's available?

Reply:
Of course.